### DISCUSSION TITLE
Monocular calibration from images/video or live video feed

### DISCUSSION THREAD

**Discussion by swishswish123** [SciKit-Surgery/scikit-surgerybard#118] [Q&A]
Hello!

what would be the best way to integrate monocular camera calibration in my program with scikit-surgery?

I have been using the bard command line app but that doesn’t seem to work for a pre-recorded video and also I would like to integrate it with my program.

I’ve been using a simple python script using an opencv tutorial but I think yours may be more accurate as I can also specify the size of the squares.

any suggestions?

**Comment by Aliflar** [comment]
Hello, regarding the question you raised, do you have an answer now? I really need the method to integrate it into other programs. Thank you.
